packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -289,8 +288,17 @@ export default class SalesforceCRMService implements CRM {
       : [];
   }
 
-  async createContacts(contactsToCreate: { email: string; name: string }[]) {
+  async createContacts(contactsToCreate: { email: string; name: string }[], organizerEmail?: string) {
     const conn = await this.conn;
+
+    // See if the organizer exists in the CRM
+    let organizerId: string;
+    if (organizerEmail) {
+      const userQuery = await this.getSalesforceUserFromEmail(organizerEmail);
+      if (userQuery) {
+        organizerId = (userQuery.records[0] as { Email: string; Id: string }).Id;
+      }
+    }
```

Comment:
See if the organizing user exists in Salesforce. If so then connect the new contact to that organizer. If they don't exist then fallback to the account that iniated the Salesforce OAuth.